strr-examiner-web/app/components/ComposeNoc.vue

```diff
@@ -0,0 +1,45 @@
+<script setup lang="ts">
+const { sendNocSchema } = useExaminerStore()
+const { nocContent, showNocModal, nocFormRef } = storeToRefs(useExaminerStore())
+const { t } = useI18n()
+
+const handleInput = () => {
+  if (nocContent.value.content.length > 1 && nocFormRef.value) {
+    nocFormRef.value.clear()
+  }
+}
+</script>
+
+<template>
+  <div v-if="showNocModal" class="app-inner-container">
+    <div class="mb-8 rounded bg-white py-6">
+      <UForm
+        ref="nocFormRef"
+        :schema="sendNocSchema"
+        :state="nocContent"
+        :validate-on="['submit']"
```

Comment:
Don't think validate-on is needed here, there is no submit button.

Reply:
If I don't include this then clicking into text box, not typing anything in content and then clicking out triggered validation error.

Reply:
All good